ap/tasks/framework/base.py

```diff
@@ -291,6 +305,38 @@ def wlcg_target(self, *path, **kwargs):
         # create the target instance and return it
         return cls(path, **kwargs)
 
+    def target(self, *path, **kwargs):
+        """ target(*path, location=None, **kwargs)
+        """
+        # get the default location
+        location = kwargs.pop("location", self.default_output_location)
+
+        # parse it and obtain config values if necessary
+        if isinstance(location, str):
+            location = OutputLocation[location]
+        if location == OutputLocation.config:
+            location = law.config.get_expanded("outputs", self.task_family, split_csv=True)
+            if not location:
+                raise Exception(
+                    f"no option 'outputs.{self.task_family}' found in law.cfg to obtain target "
+                    "location",
+                )
+            location[0] = OutputLocation[location[0]]
+        location = law.util.make_list(location)
+
+        # forward to correct function
+        if location[0] == OutputLocation.local:
+            if len(location) > 1:
+                kwargs.setdefault("store", location[1])
```

Comment:
This would be the store argument for local targets that decides in which local store directory the target would be stored. I will use a named variable here.